index.html

```diff
@@ -10,648 +10,1251 @@
-                                <span class="btn-text">Watch Video</span>
-                            </a>
+          </div>
+        </div>
+      </div>
+
+
+      <div class="container responsivegrid blurb aem-GridColumn aem-GridColumn--default--12" style="background-image:url(/assets/blurb-bg.png);">
+        <div id="container-817a18c275" class="cmp-container">
+          <div id="container" class="aem-Grid aem-Grid--12 aem-Grid--default--12 ">
+            <div class="container responsivegrid content-wrap aem-GridColumn aem-GridColumn--default--12">
+              <div id="container-c97c25b195" class="cmp-container">
+                <div id="content" class="aem-Grid aem-Grid--12 aem-Grid--default--12 ">
+                  <div class="image logo-img aem-GridColumn aem-GridColumn--default--12">
+                    <div data-cmp-lazythreshold="0" data-cmp-src="/content/leaderimpact/lang/en2/_jcr_content/content-parsys/container_291318294/container/image.coreimg{.width}.svg/1669129246299/blurb-logo.svg" data-asset="/content/dam/LeaderImpact/site-assets/blurb-logo.svg" data-title="logo" id="image-05a3573de6" class="cmp-image" itemscope="" itemtype="http://schema.org/ImageObject">
+                      <img src="assets/blurb-logo.svg" class="cmp-image__image" itemprop="contentUrl" data-cmp-hook-image="image" alt="logo">
+                    </div>
+                  </div>
+
+                  <div class="text blurb-text aem-GridColumn aem-GridColumn--default--12">
+                    <div id="text-d40dd0d09e" class="cmp-text">
+                      <p>Leaders of impact don't wait until they're older or deeper in their career to make a difference - they start today!</p>
+                    </div>
+                  </div>
+
+                  <div class="container responsivegrid aem-GridColumn aem-GridColumn--default--12">
+                    <div id="container-7666c94359" class="cmp-container">
+                      <div id="button-wrap" class="aem-Grid aem-Grid--12 aem-Grid--default--12 ">
+                        <div class="button btn-link aem-GridColumn aem-GridColumn--default--12">
+                          <a id="button-34826f1484" class="cmp-button" href="/content/leaderimpact/lang/en2/resources/impact-blog/big-dreams--small-steps.html">
+                            <span class="cmp-button__text">Learn More</span>
+                          </a>
                         </div>
+                        <div class="video-btn aem-GridColumn aem-GridColumn--default--12">
+                          <a id="button-2aa67324e3" class="cmp-button" href="/content/leaderimpact/lang/en2/resources/leading-through-a-crisis/video-series.html">
+                            <img srcset="./assets/video.svg" src="./assets/video.svg" loading="lazy" class="post-icon" itemprop="contentUrl"  alt="" title="">

```

Comment:
This would require customization. Instead, we could use the button icon feature which would look something like 
```
<div class="button video-btn">
    <a id="button-1d72cecfeb" class="cmp-button" href="/content/leaderimpact/lang/en2/resources/leading-through-a-crisis/video-series.html">
        <span class="cmp-button__icon cmp-button__icon--video-btn" aria-hidden="true"></span>
        <span class="cmp-button__text">Watch Video</span>
    </a>
</div>
```